src/components/Email/index.tsx

```diff
@@ -0,0 +1,9 @@
+import { FC, ReactNode } from 'react';
+
+interface EmailPropType {
```

Comment:
I think it's fine to define it as just `EmailProps`. React Material UI also uses the same pattern. What do you think?

Reply:
I just added `Type` with it cz it's kind of like mentioning the types for the props, that's why.
But that's fine, changing that to `EmailProps`